db/migrate/20160427110629_create_service_interactions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateServiceInteractions < ActiveRecord::Migration
+  def change
+    create_table :service_interactions do |t|
+      t.integer :service_identifier
+      t.string :service_label
+      t.integer :ineraction_identifier
```

Comment:
That said, if we do keep the service + interaction details in here, there's a typo here; it should be `interaction_identifier` but we've dropped a `t`.